lib/common/zstd_internal.h

```diff
@@ -336,6 +336,12 @@ MEM_STATIC size_t ZSTD_limitCopy(void* dst, size_t dstCapacity, const void* src,
  * In which case, resize it down to free some memory */
 #define ZSTD_WORKSPACETOOLARGE_MAXDURATION 128
 
+/* Controls whether the input/output buffer is buffered or stable. */
+typedef enum {
+    ZSTD_bm_buffered = 0,  /* Buffer the input/output */
+    ZSTD_bm_stable = 1     /* ZSTD_inBuffer/ZSTD_outBuffer is stable */
```

Comment:
The question below on the name of parameter `ZSTD_c_stableOutBuffer`
could have an impact on this name too.
In which case, `ZSTD_bm_stable` should rather insist on the _absence_ of intermediate buffer,
which is a consequence of the caller's pledge (`stable` for `input`, `largeEnough` for `output`).
`ZSTD_bm_unbuffered` comes to mind.